src/components/unique-token/UniqueTokenImage.js

```diff
@@ -102,12 +103,13 @@ const UniqueTokenImage = ({
               onError={handleError}
```

Comment:
I think we loose the fact that we're  having two images actually, lowq one and hq, small to show when the big one is loading

Reply:
how do you figure? we're still doing this as we resize to a small image via `getLowResUrl` , we can make it smaller if we need to